helm/install/templates/role.yaml

```diff
@@ -102,6 +102,29 @@ rules:
   - list
   - patch
   - watch
+- apiGroups:
+  - postgres-operator.crunchydata.com
+  resources:
+  - pgupgrades
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - postgres-operator.crunchydata.com
+  resources:
+  - pgupgrades/finalizers
+  verbs:
+  - patch
+  - update
+- apiGroups:
+  - postgres-operator.crunchydata.com
+  resources:
+  - pgupgrades/status
+  verbs:
+  - get
+  - patch
+  - watch
```

Comment:
This list should be reduced to just `- patch`.

Reply:
Huh, really? Looks like y'all made that update to postgres-operator repo too.